README.md

```diff
@@ -236,7 +196,73 @@ TODO: Add notes about how to use the system.
 
 ## 🚀 Deployment <a name = "deployment"></a>
 
-TODO: Add additional notes about how to deploy this on a live system.
+### Prerequisites
```

Comment:
Maybe here it would be good to explain that this whole section is about running the app in production mode, as opposed to the first hald that is running the app in development mode.

Also, I'm having trouble with the production mode because I have by default an apache2 server running on port 80, so maybe it would be good to say that we should shut down any such services before.

For me it was `sudo service apache2 stop`

Reply:
Updated. Thanks for the suggestions.